libmscore/tempotext.cpp

```diff
@@ -190,6 +190,7 @@ void TempoText::textChanged()
                         }
                   }
             }
+      score()->fixTicks();
```

Comment:
That fixed it. I see why it should be at that place. I'll commit these two changes.